### ISSUE TITLE
Azure ad integration

### ISSUE THREAD

**Issue by Faisal-developer** [skoruba/IdentityServer4.Admin#997]
### Question
Can anyone provide Azure AD integration example with skoruba identity server ?
### Relevant parts of the log file

```
   <log goes here>
```

**Comment by Faisal-developer** [comment]
Also if someone can provide proper documentation in order to understand how external login works.

**Comment by mosesparadise** [comment]
I think this #727 is what you are looking for.

**Comment by Faisal-developer** [comment]
@mosesparadise Yes. But don't we have more details about this. Moreover, how can we manage this for multiple clients ? I have single IDP and multiple customers are having external login.
